docs_versioned_docs/version-ros2humble/ros/config/yaml/system.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: System
+sidebar_label: System
+sidebar_position: 3
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+Proper networking setup is crucial in setting up the ROS 2 middleware and ensure other onboard computers communicate reliably.
+
+## Hosts
+
+The **hosts** section serves as a way to match IP addresses to hostnames. By default, Clearpath robots use the serial number as the hostname and have a default IP of `192.168.131.1`.
+
+- The **self** entry refers to what hostname is this configuration file on.
+- The **platform** entry refers to the hostname and IP of the robot platform's main computer.
+- The **onboard** entry is used to define the hostname and IP of other computers on the robot.
+- The **remote** entry is used to define the hostname and IP of computers in the system that are not on the robot, such as a user's PC.
+
+## ROS 2 Environment
+
+The **ros2** sections is necessary to setup the ROS 2 middleware.
+
+- **username** must match the username of the user that is to run all ROS nodes.
```

Comment:
Change to:
- **username** must match the username that is used to run all ROS nodes.